ApplensBackend/LocalDevelopmentTemplate/Detector/detector.csx

```diff
@@ -0,0 +1,19 @@
+private static string GetQuery(OperationContext<App> cxt)
+{
+    return
+    $@"<YOUR_TABLE_NAME>
```

Comment:
are you using this? We should use Product Specific templates from here:
https://github.com/Azure/Azure-AppServices-Diagnostics/tree/master/data/templates

Reply:
Oh I see.. I put it here but this will be replaced by the content of onboarding flow editor eventually, probably I need to make this content empty.

Reply:
ya.. That makes sense.. since you are in the UI, you already have the template with you.